backend/src/app.ts

```diff
@@ -173,3 +183,7 @@ function MatchRepository() {
   }
   return new InMemoryMatchRepository();
 }
+
+function TournamentTablesRepository() {
+  return new InMemoryTournamentTablesRepository();
+}
```

Comment:
Tu chyba przydałaby się inna nazwa funkcji z informacją co zwraca?

Reply:
No, zwraca repozytorium, z którego korzystamy. Jak dorobię obsługę mongo, to dodam, analogicznie, jak w innych modułach.

Reply:
Tylko zastanawiam się czy nie lepiej getTournamentTablesRepository()